tests/playwright/sanity/modules/tabs-v2/tabs-v2.test.js

```diff
@@ -1,65 +0,0 @@
-const { test, expect } = require( '@playwright/test' );
```

Comment:
Please sync with latest version of 3.10.0. This test should have been removed in the main repository aleady.

Reply:
Done